src/main/java/pl/simpleascoding/tutoringplatform/repository/ReviewRepository.java

```diff
@@ -0,0 +1,12 @@
+package pl.simpleascoding.tutoringplatform.repository;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import pl.simpleascoding.tutoringplatform.domain.review.Review;
+
+public interface ReviewRepository extends JpaRepository<Review, Long> {
+    Page<Review> findReviewsByAuthorId(Long authorId, Pageable pageable);
```

Comment:
Should be primitive `long`. Object value can be `null `and nobody want it here.

Reply:
Done